firestore/app/src/main/java/com/google/firebase/example/fireeats/kotlin/RatingDialogFragment.kt

```diff
@@ -7,18 +7,15 @@ import android.view.View
 import android.view.ViewGroup
 import androidx.fragment.app.DialogFragment
 import com.google.firebase.auth.FirebaseAuth
-import com.google.firebase.example.fireeats.R
+import com.google.firebase.example.fireeats.databinding.DialogRatingBinding
 import com.google.firebase.example.fireeats.kotlin.model.Rating
-import kotlinx.android.synthetic.main.dialog_rating.restaurantFormRating
-import kotlinx.android.synthetic.main.dialog_rating.restaurantFormText
-import kotlinx.android.synthetic.main.dialog_rating.view.restaurantFormButton
-import kotlinx.android.synthetic.main.dialog_rating.view.restaurantFormCancel
 
 /**
  * Dialog Fragment containing rating form.
  */
 class RatingDialogFragment : DialogFragment() {
 
+    private lateinit var binding: DialogRatingBinding
```

Comment:
Another fragment, we should apply the same suggestion that I had mentioned above.